btree.js

```diff
@@ -238,6 +240,13 @@ function parse(buf) {
         case '\t':
             break;
 
+        case '\r': {
```

Comment:
This is correct.  You could also simplify it with the white space fall-through above:

```
case ' ':
case '\t':
case '\r':
    break;
```

Reply:
That was my initial guess, but then I remembered that Mac (if still true) uses just '\r' for line ends.

Reply:
Classic Macs pre-Mac OS X used '\r' along with a host of other OSes but good luck running a modern browser on one of those systems.  Mac OS X today is 'nix based and uses '\n'.